.env.example

```diff
@@ -2,3 +2,5 @@ DISCORD_BOT_TOKEN=
 
 # Use ":memory:" as filepath to create the db in memory instead of creating a real file (for debugging/testing).
 DB_FILE_PATH=
+
```

Comment:
Why the empty line?

Reply:
Why not? personal preference for readability. I don't believe empty lines should be a problem or any kind of issue if its not 5 in a row